Makefile

```diff
@@ -40,7 +40,7 @@ help: ## Display this help.
 
 .PHONY: manifests
 manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
-	$(CONTROLLER_GEN) rbac:roleName=manager-role crd webhook paths="./pkg/apis/v1alpha1/;./pkg/apis/v1alpha2/" output:crd:artifacts:config=build/yaml/crd/
+	$(CONTROLLER_GEN) rbac:roleName=manager-role crd webhook paths="github.com/vmware-tanzu/nsx-operator/pkg/apis/v1alpha1;github.com/vmware-tanzu/nsx-operator/pkg/apis/v1alpha2" output:crd:artifacts:config=build/yaml/crd/
```

Comment:
Is this change related?

Reply:
if not changed, when I run "make manifests" it failed with 
/home/lxiaopei/go_projects/src/nsx-operator/bin/controller-gen rbac:roleName=manager-role crd webhook paths="./pkg/apis/v1alpha1/;./pkg/apis/v1alpha2/" output:crd:artifacts:config=build/yaml/crd/
-: main module (github.com/vmware-tanzu/nsx-operator) does not contain package github.com/vmware-tanzu/nsx-operator/pkg/apis/v1alpha1
-: main module (github.com/vmware-tanzu/nsx-operator) does not contain package github.com/vmware-tanzu/nsx-operator/pkg/apis/v1alpha2
Error: not all generators ran successfully
run `controller-gen rbac:roleName=manager-role crd webhook paths=./pkg/apis/v1alpha1/;./pkg/apis/v1alpha2/ output:crd:artifacts:config=build/yaml/crd/ -w` to see all available markers, or `controller-gen rbac:roleName=manager-role crd webhook paths=./pkg/apis/v1alpha1/;./pkg/apis/v1alpha2/ output:crd:artifacts:config=build/yaml/crd/ -h` for usage
Makefile:43: recipe for target 'manifests' failed
make: *** [manifests] Error 1